src/Compilers/VisualBasic/Portable/Symbols/Source/ImplementsHelper.vb

```diff
@@ -351,6 +351,17 @@ DoneWithErrorReporting:
                         If Not errorReported Then
                             ' Further verification of found method.
                             foundMember = ValidateImplementedMember(implementingSym, foundMember, implementedMemberSyntax, binder, diagBag)
+
+                            If Not MembersHaveMatchingTupleNames(implementingSym, foundMember) Then
```

Comment:
It turns out updating `errorReported` from `ValidateImplementedMember` did not affect existing tests.

---
In reply to: [82648462](https://github.com/dotnet/roslyn/pull/14229#discussion_r82648462) [](ancestors = 82648462,82475062)